lib/modules/manager/helmsman/__fixtures__/missingApps.yaml

```diff
@@ -0,0 +1,12 @@
+namespaces:
```

Comment:
can't we simply use an empty string as test case and inline? so we don't need this fixture?